### PR TITLE
=pro #16693 add sbt-updates to plugins.sbt

### PR DESCRIPTION
**Pull request by He-Pin**
1 change the typesafeResolver to typesafeReleases
2 add sbt-updates as a new plugin

### CODE REVIEW

File: project/plugins.sbt [hunk 1]
```diff
@@ -25,5 +25,7 @@ addSbtPlugin("com.typesafe.sbt" % "sbt-git" % "0.6.2")
 
 addSbtPlugin("pl.project13.scala" % "sbt-jmh" % "0.1.6")
 
+addSbtPlugin("com.timushev.sbt" % "sbt-updates" % "0.1.7")
```
Comment (ktoso) [inline project/plugins.sbt]: Isn't that one of these plugins better kept globally in your `~/.sbt/0.13/...` files?

Comment (hepin1989) [inline project/plugins.sbt]: Hard to say,but i think this one is out of box with minimal effort.

**Comment by 2m** [comment]
I also think this one is better in global configuration, because you can use this plugin potentially on any sbt project.

**Comment by 2m** [comment]
I am closing this. Thanks for the gathering the dependencies that need to be updated #16695.

**Comment by 2m** [comment]
Refs #16693

**Pull request closed without merging**